docker-compose.yml

```diff
@@ -1,8 +1,21 @@
 version: "3.9"
 
 services:
+#   postgres:
+#     image: postgres:10-bullseye
+#     ports:
+#       - "5433:5432"
+#     environment:
+#       POSTGRES_USER: pythondotorg
+#       POSTGRES_PASSWORD: pythondotorg
+#       POSTGRES_DB: pythondotorg
+#       POSTGRES_HOST_AUTH_METHOD: trust  # never do this in production!
+#     healthcheck:
+#       test: ["CMD", "pg_isready", "-U", "pythondotorg", "-d", "pythondotorg"]
+#       interval: 1s
+
```

Comment:
I guess we should remove this before merging, right?

```suggestion
```

Reply:
ahh yes thank you, I wasn't sure on the production deployment side what all it might entail for them to migrate.